ferveo-tdec/src/lib.rs

```diff
@@ -222,34 +244,22 @@ mod tests {
         aad: &[u8],
         ciphertext: &Ciphertext<E>,
         shared_secret: &SharedSecret<E>,
-        g_inv: &E::G1Prepared,
     ) {
         // So far, the ciphertext is valid
         let plaintext =
-            decrypt_with_shared_secret(ciphertext, aad, shared_secret, g_inv)
-                .unwrap();
+            decrypt_with_shared_secret(ciphertext, aad, shared_secret).unwrap();
         assert_eq!(plaintext, msg);
 
         // Malformed the ciphertext
         let mut ciphertext = ciphertext.clone();
         ciphertext.ciphertext[0] += 1;
-        assert!(decrypt_with_shared_secret(
-            &ciphertext,
-            aad,
-            shared_secret,
-            g_inv,
-        )
-        .is_err());
+        assert!(decrypt_with_shared_secret(&ciphertext, aad, shared_secret,)
```

Comment:
Should we just get rid of the trailing comma (one other spot on L261)?

```suggestion
        assert!(decrypt_with_shared_secret(&ciphertext, aad, shared_secret)
```

Reply:
It seems both options (with or without comma) are accepted by the linter. Anyway, I'll remove them.